<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Post3dWindowStreamlineStructuredSettingDialog</class>
 <widget class="QDialog" name="Post3dWindowStreamlineStructuredSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Streamline Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="physValLabel">
       <property name="text">
        <string>Physical Value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="solutionComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QListWidget" name="startPositionListWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="startPositionGroupBox_2">
       <property name="title">
        <string>Start Position</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="startPositionGroupBox_3">
          <property name="title">
           <string>Range</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="positionDescLabel">
               <property name="text">
                <string>Specify the area to start streamlines.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="TooltipLabel" name="areaHelpLabel">
               <property name="toolTip">
                <string>Specify the area to start streamlines, by grid indices range (I, J, K).</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../../../guibase/guibase.qrc">:/libs/guibase/images/iconQuestion.svg</pixmap>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="iminLabel">
             <property name="text">
              <string>I Min</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="SliderWithValue" name="iminSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="imaxLabel">
             <property name="text">
              <string>I Max</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="SliderWithValue" name="imaxSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="jminLabel">
             <property name="text">
              <string>J Min</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="SliderWithValue" name="jminSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="jmaxLabel">
             <property name="text">
              <string>J Max</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="SliderWithValue" name="jmaxSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="kminLabel">
             <property name="text">
              <string>K Min</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="SliderWithValue" name="kminSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="kmaxLabel">
             <property name="text">
              <string>K Max</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="SliderWithValue" name="kmaxSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="generationSpaceIntervalgroupBox">
          <property name="title">
           <string>Generation space interval</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="TooltipLabel" name="spaceHelpLabel">
               <property name="toolTip">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS UI Gothic'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:3px; margin-bottom:3px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If you specify 1, streamlines start at grid nodes.&lt;/p&gt;
&lt;p style=&quot; margin-top:3px; margin-bottom:3px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If you specify 2, streamlines start at grid nodes, and middle points of grid nodes.&lt;/p&gt;
&lt;p style=&quot; margin-top:3px; margin-bottom:3px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If you specify 1/2, streamlines start at every other grid nodes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../../../guibase/guibase.qrc">:/libs/guibase/images/iconQuestion.svg</pixmap>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="spaceLabel_minus">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSlider" name="spaceSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="spaceLabel_plus">
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="spaceValueLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>41</width>
                 <height>21</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::Panel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="margin">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="spaceMinLabel">
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>min</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="spaceMidLabel">
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>1</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_9">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="spaceMaxLabel">
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>max</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="colorLabel">
              <property name="text">
               <string>Color:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ColorEditWidget" name="colorWidget"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="widthLabel">
              <property name="text">
               <string>Width:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="TooltipLabel" name="widthHelpLabel">
              <property name="toolTip">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS UI Gothic'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:3px; margin-bottom:3px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Specify a number of pixels on display.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../../guibase/guibase.qrc">:/libs/guibase/images/iconQuestion.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="widthSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>20</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>13</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePushButton">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TooltipLabel</class>
   <extends>QLabel</extends>
   <header>guibase/tooltiplabel.h</header>
  </customwidget>
  <customwidget>
   <class>ColorEditWidget</class>
   <extends>QLabel</extends>
   <header>guibase/widget/coloreditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SliderWithValue</class>
   <extends>QSlider</extends>
   <header>guibase/widget/sliderwithvalue.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../guibase/guibase.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Post3dWindowStreamlineStructuredSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Post3dWindowStreamlineStructuredSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
